src/Pos/Ssc/GodTossing/GState.hs

```diff
@@ -81,15 +93,66 @@ instance SscGStateClass SscGodTossing where
         view gsOpenings <*>
         view gsShares
 
-loadGlobalState :: (MonadDB m, WithLogger m) => m GtGlobalState
+loadGlobalState
+    :: forall m .
+    ( MonadDB m
+    , WithLogger m
+    , HasContext LrcContext m
+    ) => m GtGlobalState
 loadGlobalState = do
     logDebug "Loading SSC global state"
-    gs <- DB.getGtGlobalState
+    gs <- DB.getGtGlobalState `catch` fallbackLoad
     gs <$ logInfo (sformat ("Loaded GodTossing state: " %build) gs)
+  where
+    fallbackLoad :: SomeException -> m GtGlobalState
+    fallbackLoad _ = do
+        logInfo $ "GtGlobalState dump isn't found, load blocks (workaround)"
+        loadGlobalStateOld
```

Comment:
Wait, after we load state we should put it into DB, shouldn't we? I understand it will be put later after we apply block, but why not do it immediately (just in case we will be stopped soon for some reason)?